target/cucumber-report.html

```diff
@@ -299,7 +299,7 @@
29-43d1-46a6-a5be-36b3849eb094","testCaseStartedId":"af405bfa-8899-4951-8978-bfc291d1282d"}},{"testStepStarted":{"timestamp":{"seconds":"1614301831","nanos":536000000},"testStepId":"4901d01d-bf57-4ff5-a971-8d3362f2b0dc","testCaseStartedId":"af405bfa-8899-4951-8978-bfc291d1282d"}},{"testStepFinished":{"testStepResult":{"status":"PASSED","duration":{"nanos":168000000}},"timestamp":{"seconds":"1614301831","nanos":704000000},"testStepId":"4901d01d-bf57-4ff5-a971-8d3362f2b0dc","testCaseStartedId":"af405bfa-8899-4951-8978-bfc291d1282d"}},{"testStepStarted":{"timestamp":{"seconds":"1614301831","nanos":704000000},"testStepId":"b6a15ec1-3200-48cc-995c-9e7f181abf54","testCaseStartedId":"af405bfa-8899-4951-8978-bfc291d1282d"}},{"testStepFinished":{"testStepResult":{"status":"PASSED","duration":{"nanos":1000000}},"timestamp":{"seconds":"1614301831","nanos":705000000},"testStepId":"b6a15ec1-3200-48cc-995c-9e7f181abf54","testCaseStartedId":"af405bfa-8899-4951-8978-bfc291d1282d"}},{"testStepStarted":{"timestamp":{"seconds":"1614301831","nanos":705000000},"testStepId":"b3a38276-92fc-4260-88a6-563a2a86deb7","testCaseStartedId":"af405bfa-8899-4951-8978-bfc291d1282d"}},{"testStepFinished":{"testStepResult":{"status":"PASSED","duration":{"nanos":1000000}},"timestamp":{"seconds":"1614301831","nanos":706000000},"testStepId":"b3a38276-92fc-4260-88a6-563a2a86deb7","testCaseStartedId":"af405bfa-8899-4951-8978-bfc291d1282d"}},{"testStepStarted":{"timestamp":{"seconds":"1614301831","nanos":706000000},"testStepId":"6e050f1e-4c42-430a-9ecc-2fb7b58a7316","testCaseStartedId":"af405bfa-8899-4951-8978-bfc291d1282d"}},{"testStepFinished":{"testStepResult":{"status":"PASSED","duration":{"nanos":17000000}},"timestamp":{"seconds":"1614301831","nanos":723000000},"testStepId":"6e050f1e-4c42-430a-9ecc-2fb7b58a7316","testCaseStartedId":"af405bfa-8899-4951-8978-bfc291d1282d"}},{"testStepStarted":{"timestamp":{"seconds":"1614301831","nanos":723000000},"testStepId":"b5c6a966-150d-4d2d-ba52-320c4328d5c4","testCaseStartedId":"af405bfa-8899-4951-8978-bfc291d1282d"}},{"testStepFinished":{"testStepResult":{"status":"PASSED","duration":{"nanos":2000000}},"timestamp":{"seconds":"1614301831","nanos":725000000},"testStepId":"b5c6a966-150d-4d2d-ba52-320c4328d5c4","testCaseStartedId":"af405bfa-8899-4951-8978-bfc291d1282d"}},{"testStepStarted":{"timestamp":{"seconds":"1614301831","nanos":725000000},"testStepId":"05a5868f-8e6f-4656-b712-9b4093e37b55","testCaseStartedId":"af405bfa-8899-4951-8978-bfc291d1282d"}},{"testStepFinished":{"testStepResult":{"status":"PASSED","duration":{"seconds":"7","nanos":410000000}},"timestamp":{"seconds":"1614301839","nanos":135000000},"testStepId":"05a5868f-8e6f-4656-b712-9b4093e37b55","testCaseStartedId":"af405bfa-8899-4951-8978-bfc291d1282d"}},{"testCaseFinished":{"timestamp":{"seconds":"1614301839","nanos":136000000},"testCaseStartedId":"af405bfa-8899-4951-8978-bfc291d1282d"}},{"testRunFinished":{"success":true,"timestamp":{"seconds":"1614301839","nanos":139000000}}}];
```

Comment:
delete and add target to gitignore, along with .iml and .idea
also, clean your local repository, then run cukesrunner to generate new clean target
https://github.com/akilahrabb/Library_Group9/pull/5/files/edb72fbe83beb045cdb0f68d697c566c0422f27f#diff-51955eacedbe86743cdf790c23b7cc21147397394d85e717de40b8c2bbbe5dbf